css/index.css

```diff
@@ -278,6 +278,13 @@ footer {
     font-size: 0.9rem;
   }
 
+  .card {
+    padding: 15px 1px;
+    width: 150%;
+    font-size: 1rem;
```

Comment:
By default the font size is 1 rem, you don't need to explicitly assign it until you have to override the font-size

Reply:
I tried it but the button overflow(last button)